src/openvic-simulation/misc/Modifier.cpp

```diff
@@ -209,24 +224,26 @@ bool ModifierManager::setup_modifier_effects() {
 	ret &= add_modifier_effect("tariff_efficiency_modifier", true);
 	ret &= add_modifier_effect("tax_eff", true);
 	ret &= add_modifier_effect("tax_efficiency", true);
+	ret &= add_modifier_effect("unemployment_benefit", true);
 	ret &= add_modifier_effect("unit_recruitment_time", false);
-	ret &= add_modifier_effect("unit_start_experience", true, RAW_DECIMAL);
 	ret &= add_modifier_effect("war_exhaustion", false);
-	ret &= add_modifier_effect("civilization_progress_modifier", true);
-	ret &= add_modifier_effect("administrative_multiplier", true);
-	ret &= add_modifier_effect("unemployment_benefit", true);
-	ret &= add_modifier_effect("minimum_wage", true);
-	ret &= add_modifier_effect("war_exhaustion_effect", false);
-	ret &= add_modifier_effect("reinforce_speed", true);
+	//<strata>_vote
+	//<strata>_income_modifier
+	//<strata>_<need>_needs
+	//<tech 'folder'>_research_bonus
+
+	/* State only*/
+	ret &= add_modifier_effect("flashpoint_tension", true);
```

Comment:
Note tension is good as you use it on cores that you don't own in the hope a crisis erupts.